3.Tests/UI/StepDefinitions/Settings/Profile/UpdateTimeZoneStepDefinition.cs

```diff
@@ -8,18 +8,23 @@ namespace SeleniumTest.Tests;
 [Scope(Feature = "Time Zone update")]
```

Comment:
I already remove this file in a new PR so you can omit these changes please.

Reply:
Done